Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -951,7 +951,7 @@ command scriptComment
    # OK-2009-03-03 : Bug 7537 - Reformat the handler after commenting, this may be flawed,
    # but the proper solution is rather time-consuming.
```

Comment:
Commenting and uncommenting multiple handlers only results in the first being formatted, which is why it should be "script" and "handler" is flawed. The proper solution ("script") is adjudged to be too time-consuming. So I think the comment and the equivalent one below should just be deleted.